src/log.h

```diff
@@ -84,6 +84,7 @@ typedef uint8_t ly_bool;
  */
 typedef enum
 {
+    LY_LLCUR = -1, /**< Special value for ::ly_log_level() which does not change the verbosity level and make it to return the current level. */
```

Comment:
Is this really a good idea? I think some other API would be better, even if a new getter would be defined.

Reply:
Well, the cons of the getter are more functions (setter and getter) and changing API. So far, `ly_log_level()` served as both getter and setter, the problem was that the getter was providing only the previous value and it was not possible to not change the value. I believe that an extra value with no effect is quite an elegant way to fill this gap.

What exactly you see as cons of this change?

Reply:
Having a bogus `LY_LOG_LEVEL` value, which is technically not a log level. I would prefer not to have it (because one must always be aware when working with this enum that there is this special value) even if it meant changing the API. Maybe simply 2 functions, one getter, one setter (having `void` return)?

Reply:
we have agreed to unify all the logger's control functions, so `LY_LLCUR` will not be added, `ly_log_dbg_groups()` will return the previous value and in all cases, it will be necessary to call the function twice to get the current value without changing it (actually by resetting it). This behavior follows the `umask()`.